moodlecli/main.py

```diff
@@ -249,11 +249,13 @@ def export_users(source_course_id, bucket_name, key):
     bucket"""
     moodle = get_moodle_client()
 
-    users = moodle.get_users_by_course(
+    user_data = moodle.get_users_by_course(
         source_course_id
     )
+    uuid_data = moodle.get_user_uuids([source_course_id])
```

Comment:
Why are we passing the course ID here?

Reply:
Agh, remnant from previous thought. ****